subworkflows/local/busco_qc.nf

```diff
@@ -5,13 +5,11 @@
 params.busco_db_options            = [:]
 params.busco_options               = [:]
 params.busco_save_download_options = [:]
-params.busco_plot_options          = [:]
 params.busco_summary_options       = [:]
 
 include { BUSCO_DB_PREPARATION            } from '../../modules/local/busco_db_preparation'        addParams( options: params.busco_db_options            )
```

Comment:
Uh, there is still `addParams`, shouldnt that be deleted by now? Those are anyway not used, correct? This is now all in `modules.config`.

Reply:
Can I just delete this or do I need to add something somewhere?

Reply:
_theoretically_ they shouldn't be used and can be deleted.

Reply:
addParams appears also in gtdbtk.nf & binning.nf & binning_preparation.nf ...